probemaker.py

```diff
@@ -213,7 +242,7 @@ def probe(self, talent: str, mod: int=0):
         elif mega_patz:
             print(self.name + ' is an idiot and mega patzed.')
```

Comment:
This method could return the result as well

Reply:
Let take care of this once we start logging, I also think we should do something about the general structure of the code